pkg/querier/queryrange/limits.go

```diff
@@ -50,10 +50,6 @@ func WithDefaultLimits(l Limits, conf queryrange.Config) Limits {
 		overrides: true,
 	}
 
-	if conf.SplitQueriesByDay {
```

Comment:
This is something I forgot to ask in my initial PR. Should we mention it in the CHANGELOG? How does CHANGELOG updating works in Loki? I was dubious.

Reply:
Good call. I can add a deprecation notice.